Manuals/Tools/Altibase_trunk/eng/Replication Manager User's Manual.md

```diff
@@ -122,83 +150,88 @@ If you need immediate assistance regarding any errors, omissions, and other tech
+- Compatible Altibase version
 
-Before the installation, please check the system requirements to install Replication Manager correctly.  The minimum system requirements for Replication Manager are as follows:
+#### Hardware Requirements
 
-1. Computer Processor: 800MHz Pentium or better
+1. CPU: 800MHz Pentium or better
 
-2. Computer Memory: 512MB or more
+2. Main Memory: 512MB or more
 
-3. Computer Disk: 50MB or more free spaces (in addition to space required for the JRE)
+3. Disk: 50MB or more free spaces (in addition to space required for the JRE)
 
 4. Screen Resolution: 1024 x 768 pixels or higher
 
-Replication Manager is a Java-based graphical client application, which means it relies on the client's hardware, operating system, and Java runtime environment. At this time, Replication Manager is now distributed in Linux-compatible packages.
+#### Software Requirements
+
+- Java Development Kit (JDK), or Java Runtime Environment (JRE) 6 or later
+
+Replication Manager is a Java application, based on Eclipse RCP 3.6.2 technology (http://www.eclipse.org/). Therefore, it will work on any platform on which Eclipse RCP is supported. For additional information, please refer to the supported platform list at http://www.eclipse.org/eclipse/development/readme_eclipse_3.6.2.html. 
 
-| Package Name                                        | Operating System | Hardware   | JRE              | Windows System |
-| --------------------------------------------------- | ---------------- | ---------- | ---------------- | -------------- |
-| ReplicationManager-linux.gtk.x86.zip (JRE included) | Linux            | x86 32-bit | Java 6 or higher | GTK            |
+At this time, Replication Manager is now distributed in Windows and Linux-compatible packages. It will be packaged for other platforms in the future if demand merits. 
 
-Replication Manager is based on Eclipse RCP 3.6.2 technology (http://www.eclipse.org/). Therefore, this version of Replication Manager will work on any platform on which Eclipse RCP is supported. For additional information, please refer to the supported platform list at http://www.eclipse.org/eclipse/development/readme_eclipse_3.6.2.html. Replication Manager will be packaged for other platforms in the future if demand merits. 
+| Package Name                                        | Operating System | Graphic System | JRE              |
+| --------------------------------------------------- | ---------------- | -------------- | ---------------- |
+| ReplicationManager-win32.win32.x86.zip              | Windows          | win32          | JRE 6 included   |
+| ReplicationManager-linux.gtk.x86.zip (JRE included) | Linux            | GTK            | Java 6 or higher |
 
-#### Installation Media
+Since Replication Manager is bundled with the JRE 6 for the Microsoft Windows system, users do not need to install the JRE. However, other operating system users must configure the JRE and related environment variables appropriately.
```

Comment:
일반적인 자바 사용을 위한 준비라 굳이 여기서 설명할 필요는 없을 것 같습니다. 문장 끝에 to use Java라고 붙이도록 하겠습니다.

그리고, Windows 패키지에만 JRE가 함께 패키징되는데 문장이 오해의 소지가 있어 Since Replication Manager for the Microsoft Windows system is bundled with the JRE 6 과 같이 어순을 바꾸겠습니다.